tools/openms/.shed.yml

```diff
@@ -0,0 +1,12 @@
+name: openms
+owner: galaxyp

```

Comment:
Hmm, I thought about that but I couldn't see a point why we would want that? Fine for me though.